lib/spack/spack/build_systems/cmake.py

```diff
@@ -267,7 +267,7 @@ def std_args(pkg, generator=None):
         args = [
             "-G",
             generator,
-            define("CMAKE_INSTALL_PREFIX", pkg.prefix),
+            define("CMAKE_INSTALL_PREFIX", sp.convert_to_posix_path(pkg.prefix)),
```

Comment:
Can we use `pathlib.Path.as_posix()`, now that we support Python 3.6+, or are there differences?